log/keyvalue_test.go

```diff
@@ -156,7 +156,7 @@ func TestFloat64(t *testing.T) {
 	v, k := kv.Value, log.KindFloat64
 	t.Run("AsBool", testErrKind(v.AsBool, "AsBool", k))
 	t.Run("AsFloat64", func(t *testing.T) {
-		assert.Equal(t, val, v.AsFloat64(), "AsFloat64")
+		assert.InDelta(t, val, v.AsFloat64(), 0.01, "AsFloat64")
```

Comment:
It will be exactly the same - this is  just transformation. If there would be a "delta" then I think it would be even a bug.